server/cloudapps/app_install.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package cloudapps
+
+import (
+	"errors"
+
+	"github.com/mattermost/mattermost-plugin-cloudapps/server/utils/md"
+)
+
+type InInstallApp struct {
```

Comment:
The idea is that each API defines its own Out type so we have very specific Go types. If some end up overlapping almost entirely I'd support merging them, otherwise keep them separate.